base_layer/wallet/src/output_manager_service/storage/sqlite_db/mod.rs

```diff
@@ -73,12 +73,55 @@ pub struct OutputManagerSqliteDatabase {
 
 impl OutputManagerSqliteDatabase {
     pub fn new(database_connection: WalletDbConnection, cipher: Option<Aes256Gcm>) -> Self {
+        // TODO: Remove this migration for the next testnet reset (currently on Dibbler) #testnet_reset
+        let _ = OutputManagerSqliteDatabase::update_json_output_features_to_v1(database_connection.clone());
```

Comment:
What about the outputs that are not v1?

Reply:
The word choice `update_json_output_features_to_v1` is bad here, it should actually read `update_json_output_features_to_include_default_recovery_byte`

Reply:
Going to replace this with

> Is it not better to just add a serde(default = 0) to the recovery_byte?